CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ include(VMCBuildMode)
 option(BUILD_SHARED_LIBS "Build the dynamic libraries" ON)
 
 #--- Find required packages ----------------------------------------------------
-find_package(ROOT REQUIRED)
+find_package(ROOT MODULE COMPONENTS Core RIO Rint Matrix Physics MathCore Thread Geom EG REQUIRED)
```

Comment:
There is no need to specify the components which are included by default; then, we also want to require CONFIG as Root provides it; so this change could be corrected as:
`find_package(ROOT CONFIG COMPONENTS Geom EG REQUIRED)`

Reply:
Yes, agreed especially given you comments below